src/analyses/base.ml

```diff
@@ -1496,7 +1498,13 @@ struct
     if eval_bool exp = Some tv then raise Deadcode
     else
       let ik = get_ikind (Cil.typeOf exp) in
-      Tuple3.map1 (fun _ -> inv_exp (ID.of_bool_ikind ik tv) exp) st
+      let tv_abs =
+        if tv
+          then IDB.of_excl_list ik [Int64.zero]
+          else IDB.of_int_ikind ik Int64.zero
+      in
+      Tuple3.map1 (fun _ -> inv_exp (tv_abs) exp) st
```

Comment:
`(tv_abs)` -> `tv_abs`

Reply:
Done, in 4976bb7a9a8f34e6fb9abe36aa3efdd04dd2902c.